.github/workflows/package-build-on-release.yml

```
@@ -0,0 +1,27 @@
+# HINT: this is a central workflow over many components and teams.
+#  on failure, ask devops.
+name: Trigger package build on release
+on:
+  release:
+    types: [released]
+
+permissions:
+  contents: read
+  packages: write
+  id-token: write
+  pull-requests: write
+
+jobs:
+  call_packaging_workflow:
+    name: Call Packaging workflow
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Trigger development-packaging workflow
+        uses: greenbone/actions/trigger-workflow@487dab5354ace3dd575c8cf100a3e522c1dd3a79
```

I don't think we need to pin our own actions to a specific commit.
```suggestion
        uses: greenbone/actions/trigger-workflow@v3
```

---

hmm, interesting point. 

While i agree that we might not need it urgently, I think it can still be a good thing, it ensures that it's fully clear which version of the workflow created a certain workflow result - for example when trying to find a problem.

And because it's automatically updated by dependabot anyway, I also see no downside of doing it. 
Am I missing something?